src/js/background/backgroundLogic.js

```diff
@@ -15,13 +15,25 @@ const backgroundLogic = {
   unhideQueue: [],
   init() {
     browser.commands.onCommand.addListener(function (command) {
-      for (let i=0; i < backgroundLogic.NUMBER_OF_KEYBOARD_SHORTCUTS; i++) {
```

Comment:
If `NUMBER_OF_KEYBOARD_SHORTCUTS` isn't needed anymore, we should remove all references to it in `backgroundLogic.js`.

Reply:
Removed `NUMBER_OF_KEYBOARD_SHORTCUTS` from `backgroundLogic.js`; however, since there was a reference to it in `identityState.js` I moved it there.